Userland/Applications/TextEditor/main.cpp

```diff
@@ -67,8 +112,33 @@ int main(int argc, char** argv)
     if (file_to_edit) {
         // A file name was passed, parse any possible line and column numbers included.
         FileArgument parsed_argument(file_to_edit);
-        if (!text_widget.open_file(parsed_argument.filename()))
-            return 1;
+        int fd;
+
+        if (!parsed_argument.filename().starts_with("/"sv)) {
```

Comment:
What is the reason you are checking for relative paths here? AFAIK, you should be able to `open()` the path relatively, even though it needs to be absolute for unveiling.

Reply:
Just a mistake, I was thinking about absolute paths too much, fixed.